engines/native/system/system_test.go

```diff
@@ -17,21 +19,81 @@ import (
 	"github.com/taskcluster/taskcluster-worker/runtime/ioext"
 )
 
+func hasString(x string, y []string) bool {
+	for _, s := range y {
+		if s == x {
+			return true
+		}
+	}
+
+	return false
+}
+
 // Note: Constants testGroup, testCat, testTrue, testFalse, testPrintDir, and
```

Comment:
I love the addition of extra tests for user groups :)

Let's document the `testGroups` variable here. Just a small thing.